libraries/helpers.rb

```diff
@@ -0,0 +1,19 @@
+module VarnishCookbook
+  # Helper methods to be used in multiple Varnish cookbook libraries.
+  module Helpers
+    def varnish_version
+      cmd = 'varnishd -V 2>&1'
+      cmd = Mixlib::ShellOut.new(cmd)
+      cmd.environment['HOME'] = ENV.fetch('HOME', '/root')
+
+      begin
+        cmd.run_command
+        Chef::Log.info "varnish version detected: #{cmd}"
+        Chef::Log.info "varnish version detected: #{cmd.stdout}"
```

Comment:
Just thinking this should be a debug-level log. And maybe just one log line for both?